### ISSUE TITLE
Eclipse has all kinds of warnings

### ISSUE THREAD

**Issue by tsember** [tsember/freight-portal#85]
that i want to eliminate.

**Comment by jpsember** [comment]
Can you paste in some of the warnings here?  Insert them within triple backtick sections such as this:

```
warning warning warning
```

**Comment by tsember** [comment]
right now there are 17 warnings:

```
Description	Resource	Path	Location	Type
ArrayList is a raw type. References to generic type ArrayList<E> should be parameterized	Tools.java	/freight-portal/src/main/java/com/tsember/basic	line 577	Java Problem
ArrayList is a raw type. References to generic type ArrayList<E> should be parameterized	Tools.java	/freight-portal/src/main/java/com/tsember/basic	line 577	Java Problem
Class is a raw type. References to generic type Class<T> should be parameterized	FileTools.java	/freight-portal/src/main/java/com/tsember/basic	line 61	Java Problem
Class is a raw type. References to generic type Class<T> should be parameterized	FileTools.java	/freight-portal/src/main/java/com/tsember/basic	line 73	Java Problem
Collection is a raw type. References to generic type Collection<E> should be parameterized	Tools.java	/freight-portal/src/main/java/com/tsember/basic	line 574	Java Problem
Collection is a raw type. References to generic type Collection<E> should be parameterized	Tools.java	/freight-portal/src/main/java/com/tsember/basic	line 581	Java Problem
Dead code	StrBuild.java	/freight-portal/src/main/java/com/tsember/basic	line 62	Java Problem
Dead code	Tools.java	/freight-portal/src/main/java/com/tsember/basic	line 71	Java Problem
List is a raw type. References to generic type List<E> should be parameterized	JSUtils.java	/freight-portal/src/main/java/com/tsember/json	line 155	Java Problem
List is a raw type. References to generic type List<E> should be parameterized	JSUtils.java	/freight-portal/src/main/java/com/tsember/json	line 173	Java Problem
List is a raw type. References to generic type List<E> should be parameterized	Tools.java	/freight-portal/src/main/java/com/tsember/basic	line 567	Java Problem
Map is a raw type. References to generic type Map<K,V> should be parameterized	JSUtils.java	/freight-portal/src/main/java/com/tsember/json	line 153	Java Problem
Map is a raw type. References to generic type Map<K,V> should be parameterized	JSUtils.java	/freight-portal/src/main/java/com/tsember/json	line 162	Java Problem
Map is a raw type. References to generic type Map<K,V> should be parameterized	Tools.java	/freight-portal/src/main/java/com/tsember/basic	line 558	Java Problem
Pair is a raw type. References to generic type Pair<T1,T2> should be parameterized	Pair.java	/freight-portal/src/main/java/com/tsember/basic	line 38	Java Problem
Pair is a raw type. References to generic type Pair<T1,T2> should be parameterized	Pair.java	/freight-portal/src/main/java/com/tsember/basic	line 38	Java Problem
The serializable class FileException does not declare a static final serialVersionUID field of type long	FileException.java	/freight-portal/src/main/java/com/tsember/basic	line 3	Java Problem
```

**Comment by jpsember** [comment]
I'm not seeing those warnings (I did a git pull and cleaned the project).  Can you try cleaning your project and see if it goes away?

**Comment by tsember** [comment]
i did that but let me try again. what i havent done is the "rake eclipse"
stuff, so let me revisit that...


i will let you know how it goes...

**Comment by tsember** [comment]
rake eclipse

solved it. sorry...

**Comment by tsember** [comment]
and now all the warnings are back after i did a "git pull". and "rake eclipse" , closing and opening project in Eclipse, and cleaning project in Eclipse have no effect.

So there is some setup issue on my machine, but i think is totally a local issue on my machine...

**Comment by tsember** [comment]
These errors are exactly the same as before, but now they never go away, no matter cleaning, closing and reopening Eclipse, etc.

and i need to solve it because it is not immediately obvious to me when i introduce compiler warnings, so i can't tell that i have done so...
